src/main/java/com/thinkful/noteful/folders/Folder.java

```diff
@@ -16,81 +21,87 @@
-import com.thinkful.noteful.notes.Note;
-import com.thinkful.noteful.users.User;
+
 
 @Entity
 @EntityListeners(AuditingEntityListener.class)
-@JsonIgnoreProperties(value={"createdAt", "updatedAt"}, allowGetters = true)
+@JsonIgnoreProperties(value = {"createdAt", "updatedAt"}, allowGetters = true)
 public class Folder {
 
-    @Id
-    @GeneratedValue( strategy = GenerationType.AUTO)
-    private Long id;
+  public Folder() {}
+
+  public Folder(Long id) {
+    this.id = id;
+  }
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private Long id;
+
+  @NotBlank
+  private String name;
 
-    @NotBlank
-    private String name;
+  @OneToMany(mappedBy = "folder")
+  @JsonIgnore
+  private List<Note> notes;
 
-    @OneToMany(mappedBy="folder")
-    private List<Note> notes;
+  @ManyToOne
+  @JoinColumn(name = "userid")
+  @JsonAlias({"userId"})
+  private User user;
 
-    @ManyToOne
-    @JoinColumn(name="userid")
-    @JsonAlias({"userId"})
-    private User user;
+  @Column(nullable = false, updatable = false)
+  @Temporal(TemporalType.TIMESTAMP)
+  @CreatedDate
+  private Date createdAt;
 
-    @Column(nullable = false, updatable = false)
-    @Temporal(TemporalType.TIMESTAMP)
-    @CreatedDate
-    private Date createdAt;
+  @Column(nullable = false)
+  @Temporal(TemporalType.TIMESTAMP)
+  @LastModifiedDate
+  private Date updatedAt;
 
-    @Column(nullable = false)
-    @Temporal(TemporalType.TIMESTAMP)
-    @LastModifiedDate
-    private Date updatedAt;
+  public Long getId(){
+    return this.id;
+  }
 
-    public String getName(){
-        return this.name;
-    }
+  public String getName() {
+    return this.name;
+  }
 
-    public void setName(String name){
-        this.name = name;
-    }
+  public void setName(String name) {
+    this.name = name;
+  }
 
-    public Date getCreatedAt(){
-        return this.createdAt;
-    }
+  public Date getCreatedAt() {
+    return this.createdAt;
+  }
 
-    public void setCreatedAt(Date createdAt){
-        this.createdAt = createdAt;
-    }
+  public void setCreatedAt(Date createdAt) {
+    this.createdAt = createdAt;
+  }
 
-    public Date getUpdatedAt(){
-        return this.updatedAt;
-    }
+  public Date getUpdatedAt() {
+    return this.updatedAt;
+  }
 
-    public void setUpdatedAt(Date updatedAt){
-        this.updatedAt = updatedAt;
-    }
+  public void setUpdatedAt(Date updatedAt) {
+    this.updatedAt = updatedAt;
+  }
 
-    public List<Note> getNotes(){
-        return this.notes;
-    }
+  public List<Note> getNotes() {
+    return this.notes;
+  }
 
-    public void setNotes(List<Note> notes){
-        this.notes = notes;
-    }
+  public void setNotes(List<Note> notes) {
+    this.notes = notes;
+  }
 
-    public User getUser(){
-        return this.user;
-    }
+  public User getUser() {
+    return this.user;
+  }
 
-    public void setUser(User user){
-        this.user = user;
-    }
+  public void setUser(User user) {
+    this.user = user;
+  }
```

Comment:
Hibernate and Domain side look great!

![capture](https://user-images.githubusercontent.com/10905039/39965054-b0552bca-5646-11e8-87c9-5283a4357d82.PNG)